README.md

```diff
@@ -82,6 +82,11 @@ The ``--no-use-reorder`` option prevents the fixer from re-ordering USE statemen
 $ symfony-upgrade-fixer fix /path/to/code --no-use-reorder
 ```
 
+The ``--fixers`` option allows to specify which fixers should be used:
+```bash
+$ symfony-upgrade-fixer fix /path/to/code --fixers=get_request,progress_bar,property_access
```

Comment:
Let's add some of our upgrade fixers as an example, this are PHP-CS fixers I believe.

Reply:
@umpirsky I took this fixers from the next list:
https://github.com/umpirsky/Symfony-Upgrade-Fixer#fixers-available

As I understand it's a fixers from this project: https://github.com/umpirsky/Symfony-Upgrade-Fixer/tree/master/src/Symfony/Upgrade/Fixer.

But anyway, I have changed `progress_bar` on `form_events`

Reply:
@aivus Sorry, long time ago I worked on this, already forgetting all the fixers. :)